simba/report.py

```diff
@@ -267,6 +291,268 @@ def generate(schedule, scenario, args):
     logging.info(f"Plots and output files saved in {args.results_directory}")
 
 
+def bus_type_distribution_route_rotation(extended_plots_path, schedule):
+    """Plots the distribution of bus types in consumption brackets as a stacked bar chart.
+
+    :param extended_plots_path: Path argument which combines args.results_directory
+                                and string 'extended_plots'
+    :type extended_plots_path: PosixPath
+    :param schedule: Driving schedule for the simulation. schedule.rotations are used
+    :type schedule: eBus-Toolbox.Schedule
+    """
+
+    step = 50
+    max_route = int(max([schedule.rotations[rot].distance / 1000. for rot in schedule.rotations]))
```

Comment:
why are you giving 1000 as float, but then cast to int anyway? I think the decimal point can be removed here (also in l. 314)